src/resolve/mod.rs

```diff
@@ -257,8 +196,8 @@ mod tests {
     fn domain_resolves() {
```

Comment:
This test needs to be reworked. Remove the `struct` and `add_provider` call then the builtin hardcoded mechanism should take over.

Reply:
Without regards to the review content, is it possible to tag multiple and existing lines in a review? That would have made the comment more helpful.

Reply:
Not for a review comment directly, but within the review comment you can use [this](https://github.community/t5/How-to-use-Git-and-GitHub/PR-multiline-review/m-p/6779#M2261).